roles/ansible-test/tasks/main.yaml

```diff
@@ -28,9 +28,17 @@
   import_tasks: split_targets.yaml
   when: ansible_test_split_in > 1
 
+- name: Erase coverage report
+  args:
+    chdir: "{{ _test_location }}"
+    executable: /bin/bash
+  environment: "{{ ansible_test_environment | default({}) }}"
+  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_coverage_erase }}"
+  when: ansible_test_test_command == 'units'
+
```

Comment:
lets move this into a post-run playbook for units, eg:  playbooks/ansible-test-units-base/post.yaml

then add the playbook below to the ansible-test-units-base job